lib/nexus/nexus_plot.cpp

```diff
@@ -255,20 +263,56 @@ ecl_sum_type* nex::ecl_summary(const std::string& ecl_case, const NexusPlot& plt
 
      */
 
-    auto *smspec = ecl_sum_add_var(ecl_sum, "FOPR", NULL, -1, "x", 0.0);
+
+
+     /*
+      * Write Field
+      */
+
+    std::map< std::string, std::string > kw_nex2ecl {
+        { "QOP ", "FOPR" },
+        { "QWP ", "FWPR" },
+        { "QGP ", "FGPR" },
+        { "GOR ", "FGOR" },
+        { "WCUT", "FWCT" },
+        { "PAVT", "FPR " },
+        { "COP ", "FOPT" },
+        { "CWP ", "FWPT" },
+        { "CGP ", "FGPT" },
+        { "QWI ", "FWIR" },
+        { "QGI ", "FGIR" },
+        { "CWI ", "FWIT" },
+        { "CGI ", "FGIT" },
+        { "QPR ", "FCPR" },
+        { "CPP ", "FCPC" },
+        { "CTP ", "FTPR" },
+        { "TC  ", "FTPC" }
+    };
+
+    auto field_class_vars = plt.varnames("FIELD   ");
+    std::vector< std::string > field_vars;
+    std::vector< smspec_node_type* > smspec_nodes;
+    for (const auto& kw : field_class_vars) {
+        auto it = kw_nex2ecl.find(kw);
+        if (it != kw_nex2ecl.end()) {
+            field_vars.push_back(kw);
+            const auto& ecl_kw = it->second;
+            auto* node = ecl_sum_add_var( ecl_sum, ecl_kw.c_str(), NULL, -1,
+                                          "x", 0.0 );
+            smspec_nodes.push_back(node);
+        }
```

Comment:
```C++
else
     std::cerr << "Do not know how to translate " << kw << " to eclipse" << std::endl;
```

Alternatively - if the `if (it != kw_nex2ecl.end())` should *never* fail - I suggest we rather use `at` - which signals exactly that.

Reply:
This can fail without being wrong. There are some of variables in nexus not supported by eclipse. These could however be specified in eclupse by using user-defined quantities, for example `FUWOR` (Field, user-defined, water, oil, ratio).

Reply:
I see - then; I suggest we stick to a `std::cerr` warning if the test fails.